makefont/src/main.rs

```diff
@@ -0,0 +1,68 @@
+use std::env::args;
+use std::fs::write;
+use std::process::exit;
+use image::GenericImageView;
+
+const CHARACTER_WIDTH: u32 = 9;
+const CHARACTER_HEIGHT: u32 = 8;
+
+fn main() {
+	let args: Vec<String> = args().collect();
```

Comment:
I think it would be nicer to use an arg-parsing library. I'll let you pick whichever you prefer.